README.md

```diff
@@ -1 +1,177 @@
 # Knot.x Knot Engine
+
+# How does it works
+It is a handler in server flow...
+
+# How to configure
+Knot Engine processes **all** fragments containing `flowAlias` or `flow` configuration 
+entries. This section describes how flow can be defined in fragment configuration. The last chapter
+explains how Knot Engine can be configured in [Knot.x Server](https://github.com/Knotx/knotx-server-http).
+
+## Flow configuration
+Knot Engine allows to pre-configure flows and it steps.
+
+### Pre-configured flows
+The fragment configuration looks like
+```json
```

Comment:
This is unclear. What fragment configuration? Maybe some snippet with fragment here?